portfolio/src/main/webapp/script.js

```diff
@@ -29,25 +29,36 @@ function addRandomQuote() {
   quoteContainer.innerText = quote;
 }
 
+// Retrieves the comments from Datastore.
 function getComments() {
   fetch('/data').then(response => response.json()).then((comments) => {
- 
+      
```

Comment:
Random spaces here, I assume that's an error. Doesn't actually affect anything, I'm just surprised that the cloud editor doesn't automatically autoformat that away.